### ISSUE TITLE
Add Antique / XP lamps / Certificates / other consumable rewards handling 

### ISSUE THREAD

**Issue by Dreambotter420** [NotSinan/DreamBotQuester#63]
It seems like a good idea to me to have a generic leaf for this, with settings in GUI to set how to handle them

**Comment by NotSinan** [comment]
Yeah sounds like a good idea. User should be able to select which skill they go onto.

Might not really be required if we do not implement a quest scheduler.

Exit script if any consumables that are not bankable in inventory, probably? GUI alerts user that cannot start as consumable that cannot be banked is in the inventory.
